api/RTCIceCandidate.json

```diff
@@ -134,10 +134,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
Yep, https://trac.webkit.org/changeset/266698/webkit added `address` and other attributes recently, they haven't shipped yet.